### PR TITLE
cancel action added

### PR DESCRIPTION
**Pull request by abhishek-rr**

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -178,6 +178,29 @@ def test_delete_order(self):
...
+        """ change the status of an order to cancelled """
+        # create an Action to change order status to canceled
+        order_item = OrderItem(product_id=1, quantity=1, price=5, order_id = 1)
+        order = Order(customer_id=1, tracking_id=1, status=OrderStatus.Created, order_items=[order_item])
+        resp = self.app.post(
+            "/order", 
+            json=order.serialize(), 
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
```
Comment (shantanu-dahiya) [inline tests/test_routes.py:191]: Add an assert to ensure the order id is 1.

File: tests/test_routes.py [hunk 2]
```diff
@@ -178,6 +178,29 @@ def test_delete_order(self):
...
+
+        # Action to cancel the order
+        #new_order = resp.get_json()
+        #new_order["status"] = OrderStatus.Cancelled
+        resp = self.app.put(
+            "/order/1/cancel"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        updated_order = resp.get_json()
```
Comment (shantanu-dahiya) [inline tests/test_routes.py:200]: We should not check the response json here, we should fetch the order from the database again and check the status on that.

**Review by shantanu-dahiya — CHANGES_REQUESTED** [review]

File: orders_postman_collection.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
```
Comment (shantanu-dahiya) [inline orders_postman_collection.json:1]: Why are there so many deletions in this file? Is it intentional? If not, please remove these changes from this PR.

**Review by shantanu-dahiya — CHANGES_REQUESTED** [review]

File: orders_postman_collection.json (hunk 3 above)
Comment (abhishek-rr) [reply orders_postman_collection.json:1]: So the thing is, I had imported the file to postman -> added the extra action request -> exported it.
Thus, I had to replace the original file which Git realizes as making individual changes to each line.

File: tests/test_routes.py (hunk 2 above)
Comment (abhishek-rr) [reply tests/test_routes.py:200]: oh alright, will do that!

File: tests/test_routes.py (hunk 1 above)
Comment (abhishek-rr) [reply tests/test_routes.py:191]: Got it!

**Review by shantanu-dahiya — APPROVED** [review]

**Pull request merged**